tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/UserAPI.java

```diff
@@ -39,6 +46,7 @@ public UserInfo(int userid) {
             this.org = null;
             this.emailHash = null;
             this.name = "User #" + userid;
+            this.claSigned = false; // TODO
```

Comment:
TODO should have ticket link; and it's not clear what needs to be done, if anything; false makes sense for a logged-out user